src/ploomber/tasks/notebook.py

```diff
@@ -609,16 +625,43 @@ def __init__(
         self.check_if_kernel_installed = check_if_kernel_installed
         self.debug_mode = debug_mode
 
-        if "cwd" in self.papermill_params and self.local_execution:
+        # We are migrating to nb_executor_params
+        if self.papermill_params != {}:
```

Comment:
you can simplify this condition with if `self.papermill_params`, same for the others